### PR TITLE
remove win-textfile script

### PR DESCRIPTION
**Pull request by rfomin**
There is also a convert-icon script. I'm thinking about writing a python script to generate a documentation (maybe a wiki page?)/man-file, like in Chocolate Doom.

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
> There is also a convert-icon script. 

This is only ever needed when the icon is changed. I don't think it will have to be part of the build system.

**Comment by fabiangreffrath** [comment]
There is also another Python script in textscreen which is used to regenerate the fonts.

File: CMakeLists.txt [hunk 1]
```diff
@@ -172,6 +165,8 @@ else()
     install(FILES betagrph.wad DESTINATION "share/${PROJECT_TARNAME}")
     install(FILES betalevl.wad DESTINATION "share/${PROJECT_TARNAME}")
     install(FILES watermap.wad DESTINATION "share/${PROJECT_TARNAME}")
+    install(DIRECTORY docs/ DESTINATION "share/${PROJECT_TARNAME}/docs")
```
Comment (fabiangreffrath) [inline CMakeLists.txt:168]: Destination `share/doc/${PROJECT_TARNAME}`, please.

**Comment by rfomin** [comment]
> There is also another Python script in textscreen which is used to regenerate the fonts.

I think we should keep the convert-* scripts but remove them from the build system.

**Comment by fabiangreffrath** [comment]
Yes, right, this is what I meant.

File: CMakeLists.txt [hunk 2]
```diff
@@ -172,6 +165,8 @@ else()
     install(FILES betagrph.wad DESTINATION "share/${PROJECT_TARNAME}")
     install(FILES betalevl.wad DESTINATION "share/${PROJECT_TARNAME}")
     install(FILES watermap.wad DESTINATION "share/${PROJECT_TARNAME}")
+    install(DIRECTORY docs/ DESTINATION "share/docs/${PROJECT_TARNAME}")
```
Comment (fabiangreffrath) [inline CMakeLists.txt:168]: share/doc - without 's' 😉

**Comment by fabiangreffrath** [comment]
It's amazing how this single step simplifies the entire build system!

**Pull request merged**